From: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
Date: Fri, 7 Mar 2014 19:48:25 -0300
Subject: [PATCH 003/117] IB/mlx5_core: remove unreachable function call in
 module init
Patch-mainline: v3.15-rc1
Git-commit: c120e9e03090b4f9578ca38ef4250ff3805b6e3f
References: bsc#923036 FATE#318772

The call to mlx5_health_cleanup() in the module init function can never
be reached. Removing it.

Signed-off-by: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
Acked-by: Eli Cohen <eli@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Hadar Hen Zion <hadarh@mellanox.com>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/mellanox/mlx5/core/main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/mellanox/mlx5/core/main.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/main.c
@@ -532,7 +532,6 @@ static int __init init(void)
 
 	return 0;
 
-	mlx5_health_cleanup();
 err_debug:
 	mlx5_unregister_debugfs();
 	return err;
